CMakeLists.txt

```diff
@@ -291,6 +292,8 @@ else()
   message(STATUS "Not found OpenMP")
 endif()
 
+
```

Comment:
Any need for multiple empty lines?

Reply:
No, its still a draft :-) getting things wprking, then I'll clean it up.